tests/unit/lib/charms/vault_k8s/v0/test_vault_autounseal.py

```diff
@@ -245,22 +230,16 @@ def test_given_1_outstanding_and_1_satisfied_request_when_get_outstanding_reques
             secrets=[vault_autounseal_relation_1_credentials_secret],
             leader=True,
         )
-        action = scenario.Action(
-            "get-outstanding-requests",
-        )
-
-        action_output = self.ctx.run_action(action, state_in)
-
-        assert action_output.success is True
-        assert action_output.results
-        assert action_output.results["relations"] == [vault_autounseal_relation_2.relation_id]
+        self.ctx.run(self.ctx.on.action("get-outstanding-requests"), state_in)
+        assert self.ctx.action_results
+        assert self.ctx.action_results["relations"] == [vault_autounseal_relation_2.id]
 
     def test_given_satisfied_request_when_get_outstanding_requests_then_request_is_not_returned(
         self,
     ):
         vault_autounseal_relation_credentials_secret = scenario.Secret(
             id="0",
```

Comment:
I believe we don't need to set id's in secrets anymore

Reply:
Removed, except in `test_given_all_details_present_when_get_details_then_details_are_returned` where we need to mock the exact ID as we have it in peer relation data and is used to fetch the secret by the charm code